doc/ref_cert/RC2/chapters/chapter03.md

```diff
@@ -2,3 +2,25 @@
 
 # 3. Kubernetes Test Cases and Traceability to CNTT Requirements
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
+
+## Introduction
+
+All of the requirements for RC2 have been defined in the Reference Model (RM) and Reference Architecture (RA2). The scope of this chapter is to identify and list down test cases based on these requirements. Users of this chapter will be able to use it to determine which test cases they must run in order to test compliance with the requirements. This will enable traceability between the test cases and requirements. They should be able to clearly see which requirements are covered by which tests and the mapping from a specific test result (pass or fail) to a requirement. Each requirement may have one or more test case associated with it.
+
+### Goals
+- Clear mapping between requirements and test cases
+- Provide a stable set of point-in-time requirements and tests to achieve conformance
+- Enable clear tracability with versioning to know which requirements have and haven't been covered and track changes over time
```

Comment:
I assume this refers to the various versions of the RC document, i.e., the traceability is enabled by comparing different versions of RC documents. This statement does not imply that the RC document itself contains such historic data. This would also contradict the previous point a bit.

Reply:
I was reading this as each RC document has traceability to one set of point in time requirements while the RC document can evolve from version to version. Is there a way we could write this more clearly?

Reply:
How about something along the lines of
```suggestion
- Enable clear traceability of the coverage of requirements across consecutive releases of this document
```